package/npm_dependencies.js

```diff
@@ -0,0 +1,21 @@
+const { spawn } = require("child_process");
+
+// FIXME seems unused?
```

Comment:
This was a wrapper I did for formality. We can remove it

Reply:
ok will do